stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/event/handler/internal/EventBatchUpdate.kt

```diff
@@ -124,7 +124,8 @@ internal class EventBatchUpdate private constructor(
     suspend fun execute() {
         // actually insert the data
         currentUserId?.let { userMap -= it }
-        enrichChannelsWithCapabilities()
+        // TODO delete if regression goes well
+        // enrichChannelsWithCapabilities()
```

Comment:
### Fix 4
We don't need this, since we 
- fetch all channels from DB right before applying event changes
- merge `event.channel`  partially to keep other valid fields

More details about current `enrichChannelsWithCapabilities` logic below in code comments